script/GenKey.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# usage: ./GenKey.sh [<directory_name>]
```

Comment:
Let's rename the script to `gen_keys.sh` and I would also put a big warning comment up here that this script should not be used for generating keys for production usage.